integrationtest/docker/jdk8/Dockerfile

```diff
@@ -12,4 +12,6 @@ WORKDIR $SUT_HOME
 ADD start.sh /opt/start
 RUN chmod +x /opt/start
 
+COPY storm.yaml SUT_HOME
```

Comment:
No need to add a real file to Docker image. You have already create one on the fly using `echo content > file`